README.md

```diff
@@ -1,6 +1,8 @@
 OpenHMIS
 ========
 
+Current build status:  ![Build status](https://travis-ci.org/cecilia-donnelly/OpenHMIS.svg?branch=master)
```

Comment:
As we discussed, can guess-edit this to `hmis-tools/hmis-api-server.svg` before merging.